sdcm/nemesis.py

```diff
@@ -259,6 +259,11 @@ def get_node_info_list(verification_node):
                 # Replace the node that was terminated.
                 if add_node:
                     self._add_and_init_new_cluster_node()
+                # after decomission and add_node, the left nodes have data that isn't part of their tokens anymore.
+                # In order to eliminate cases that we miss a "data loss" bug because of it, we cleanup this data.
+                for node in self.cluster.nodes:
+                        # TODO: wherever we have keyspace1 hardcoded, to replace it with a list of keyspaces.
```

Comment:
@amoskong, I removed the part that verify the node isn't the new node or the target node because it's not correct.
The target_node isn't part of the list in that point + new_node may be missing when not doing add_node.
anyway nodetool cleanup should't cause any issue to run on the new node.